app/src/components/main/modal/darkModalLayout/darkModalLayout.jsx

```diff
@@ -80,34 +82,40 @@ export const DarkModalLayout = ({
   }, [isCtrlEnterPress]);
 
   return (
-    <div className={cx('container')} onClick={handleClickOutside}>
-      <CSSTransition
-        timeout={200}
-        in={isShown}
-        classNames={cx('window-animation')}
-        onExited={() => dispatch(hideModalAction())}
-      >
-        {(status) => (
-          <div ref={wrapperRef} className={cx('wrapper')}>
-            <div className={cx('layout')}>
-              <ModalHeader
-                text={title}
-                onClose={closeModalWindow}
-                renderHeaderElements={renderHeaderElements}
-              />
-              <ScrollWrapper hideTracksWhenNotNeeded autoHide>
-                {status !== 'exited' ? children : null}
-                <div className={cx('note-row')}>
-                  {modalNote && clickOutside && modalHasChanges && (
-                    <ModalNote message={modalNote} icon={ErrorInlineIcon} status={'error'} />
-                  )}
-                </div>
-              </ScrollWrapper>
+    <CSSTransition
+      timeout={300}
+      in={isOpen}
+      classNames={cx('window-animation')}
+      onExited={() => dispatch(hideModalAction())}
+    >
+      {(status) => (
+        <div className={cx('modal-content')}>
+          <div
+            className={cx('container', { 'shown-more': !isShownLess })}
+            onClick={handleClickOutside}
+          >
+            <div ref={wrapperRef} className={cx('wrapper', { 'shown-more': !isShownLess })}>
```

Comment:
You use the opposite value everywhere for `isShownLess`.
Why not provide the `**showMore**` property directly to this component with the corresponding opposite value?